apps/e2e/modals-storybook/src/app/modal/modal.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component } from '@angular/core';
+import {
+  SkyModalConfigurationInterface,
+  SkyModalInstance,
+  SkyModalService,
+} from '@skyux/modals';
+
+import { ModalBasicComponent } from './modals/modal-basic.component';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.scss'],
+})
+export class ModalComponent {
+  public buttonsHidden = false;
+
+  constructor(private modal: SkyModalService) {}
+
+  public hideButtons(): void {
+    this.buttonsHidden = true;
+  }
+  public showButtons(): void {
+    this.buttonsHidden = false;
+  }
+
+  public onOpenModalClick(): void {
+    this.openModal(ModalBasicComponent, {
+      providers: [],
```

Comment:
You should be able to leave out the `providers` array if it's empty, and if it's otherwise the only property on the `options` object, you can leave out the parameter altogether. Apply this to the rest of the code in this file.

`this.openModal(ModalBasicComponent);`